WebDriverAgentLib/Commands/FBCustomCommands.m

```diff
@@ -227,4 +228,43 @@ + (BOOL)isKeyboardPresent {
   return FBResponseWithOK();
 }
 
++ (id<FBResponsePayload>)handleGetDeviceInfo:(FBRouteRequest *)request
+{
+  // Returns locale like ja_EN and zh-Hant_US. The format depends on OS
+  // Developers should use this locale by default
+  // https://developer.apple.com/documentation/foundation/nslocale/1414388-autoupdatingcurrentlocale
+  NSString *currentLocale = [[NSLocale autoupdatingCurrentLocale] localeIdentifier];
+
+  return
+  FBResponseWithStatus(
+    FBCommandStatusNoError,
+    @{
+      @"currentLocale": currentLocale,
+      @"timeZone": [self getTimeZone],
+      }
+    );
+}
+
+/**
+ * @return The string of TimeZone. Returns TZ timezone id by default. Returns TimeZone name by Apple if TZ timezone id is not available.
+ */
++ (NSString *)getTimeZone
```

Comment:
the usual naming practices in objc don't suggest to add `get` prefix to such methods. `+ (NSString *)timeZone` should be fine

Reply:
thanks, get/set prefix confused me in various lang..